eland/ml/ml_model.py

```diff
@@ -380,6 +376,120 @@ def import_model(
         >>> # Delete model from Elasticsearch
         >>> es_model.delete_model()
         """
+
+        return cls._import_model(
+            es_client=es_client,
+            model_id=model_id,
+            model=model,
+            feature_names=feature_names,
+            classification_labels=classification_labels,
+            classification_weights=classification_weights,
+            es_if_exists=es_if_exists,
+        )
+
+    @classmethod
+    def import_ltr_model(
+        cls,
+        es_client: Union[str, List[str], Tuple[str, ...], "Elasticsearch"],
+        model_id: str,
+        model: Union[
+            "DecisionTreeRegressor",
+            "RandomForestRegressor",
+            "XGBRanker",
```

Comment:
Are `XGBRanker` (and possibly other model types) meant to be imported only with this function? If so, consider removing the type from `import_model()`'s `model` property, since that function doesn't support passing in an LTR `inference_config` anymore.

Reply:
We can imagine one that want to use a ranker model in a non-LTR context, so I would recommend to keep it supported in the regular import_model method.